server/swap/swap.go

```diff
@@ -1974,26 +1786,13 @@ func (s *Swapper) processRedeem(msg *msgjson.Message, params *msgjson.Redeem, st
 	log.Debugf("processRedeem: sending 'redemption' request to counterparty %v (%s) "+
 		"for match %v", ack.user, makerTaker(ack.isMaker), matchID)
 
-	// Register that there is an outstanding ack request. This is unregistered
-	// on completion of processAck or request expiry.
-	s.setLiveAcker(notification, ack)
-
-	// Expire function to unregister the outstanding request.
-	expireFunc := func() {
-		s.rmLiveAckers(notification.ID)
-		log.Infof("Timeout waiting for 'redemption' request from user %v (%s) for match %v",
-			ack.user, makerTaker(ack.isMaker), matchID)
-	}
-
 	// Send the ack request.
 
-	// The counterparty does not need to actually locate the redemption txn, but
-	// they do need to author and broadcast their own redeem, so allow the
-	// request to wait for a response up until the inaction penalty.
-	s.authMgr.RequestWhenConnected(ack.user, notification, func(_ comms.Link, resp *msgjson.Message) {
+	// The counterparty does not need to actually locate the redemption on txn,
+	// so use the default request timeout.
+	s.authMgr.Request(ack.user, notification, func(_ comms.Link, resp *msgjson.Message) {
 		s.processAck(resp, ack) // resp.ID == notification.ID
-	}, s.bTimeout, auth.DefaultConnectTimeout, expireFunc)
-	// maybe: s.bTimeout-time.Since(redeemTime) or time.Until(redeemTime.Add(s.bTimeout))
+	})
```

Comment:
We have seen that this can be too short given the operations the client must perform, but we can try going back to default response timeout.  Although if we really stop requiring the acks, it's fine.

Reply:
I'll do a `RequestWithTimeout` here.